addon/@ember/test-waiters/waiter-manager.ts

```diff
@@ -5,20 +5,21 @@ import { registerWaiter } from '@ember/test';
 
 type Indexable = Record<any, unknown>;
 
+// NOTE: We can't use symbols when we could have multiple copies of the addon
+//       in the dep tree. So for now, we can use this private string-based key
+//       that all copies of the `@ember/test-waiters` can use.
+const PRIVATE_GLOBAL_DATA_KEY =
+  '__#private_global_state_for_test_waiters__do_not_access_if_you_want_your_projects_to_keep_working__';
+
 // this ensures that if @ember/test-waiters exists in multiple places in the
 // build output we will still use a single map of waiters (there really should
 // only be one of them, or else `settled` will not work at all)
 const WAITERS: Map<WaiterName, Waiter> = (function () {
-  const HAS_SYMBOL = typeof Symbol !== 'undefined';
```

Comment:
symbols are unique per instantiation, so we can't actually rely on them to be accessible between multiple copies of test-waiters

Reply:
What about https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for?

Reply:
Yeah, reading the code more carefully, it seems like it is already doing what you are describing, why isn’t it working?

Reply:
thanks, updated!